css/custom.css

```diff
@@ -2567,6 +2567,11 @@ footer {
     background-position: 0 -13rem;
 }
 
+.download-platforms h2 span.icon-dragonflybsd {
```

Comment:
This isn't correct, it still shows the FreeBSD logo. I had a quick look through Google Images but I couldn't find a suitable logo to put in the spritesheet.